.github/workflows/root-ci-config/build_utils.py

```diff
@@ -76,6 +76,14 @@ def print(self) -> None:
                 print(f"""\
 Grab the image:
 $ docker run {' '.join(self.docker_opts)} -it {self.image}
+Setup the Python virtual environment after checking out root and roottest (see below)
+$ mkdir <ROOT build dir>
+$ cd <ROOT build dir>
```

Comment:
Isn't this already done in the later instructions?

Reply:
correct, my bad.